spark/v3.3/spark/src/main/scala/org/apache/spark/sql/execution/datasources/SparkExpressionConverter.scala

```diff
@@ -37,14 +37,15 @@ object SparkExpressionConverter {
   }
 
   @throws[AnalysisException]
```

Comment:
Nit; should we remove this?

Reply:
yes. we should remove it since this function does not raise such exceptions any more. 
let me do it now.